general/auth/google.rst

```diff
@@ -1,49 +1,116 @@
 
-.. image:: media/google06.png
-    :align: center
+.. _google/oauth:
 
-.. image:: media/google07.png
-    :align: center
+OAuth consent screen
+~~~~~~~~~~~~~~~~~~~~
 
-- Check that the application type is set on **Web Application**. Now configure the allowed pages on which you will be redirected.
+#. On the left side menu, click on :menuselection:`OAuth consent screen`.
 
-- To achieve this, complete the field **Authorized redirect URIs**. Copy paste the following link in the box: http://mydomain.odoo.com/auth_oauth/signin. Then click on **Create**
+   .. image:: media/google-oauth-consent-selection.png
+      :align: center
+      :alt: Google oauth consent selection menu
 
-.. image:: media/google08.png
-    :align: center
+#. Choose one of the options **(Internal / External)** as instructed, and click on *Create*.
 
-.. image:: media/google09.png
-    :align: center
+   .. image:: media/google-oauth-consent.png
+      :align: center
+      :alt: Choice of a user type in oauth consent
 
-- Once done, you receive two information (your Client ID and Client Secret). You have to insert your Client ID in the **General Settings**.
+#. Fill out your details and domain info, then click on *Save and Continue*.
+
+   .. image:: media/google-auth-edit-app-registration.png
+      :align: center
+      :alt: Edit app registration page
+
+#. Once on the **Scopes** page, leave all fields as is, and click on *Save and Continue*
+
+.. _google/credentials:
+
+Credentials
+~~~~~~~~~~~
+
+#. On the left side menu, click on :menuselection:`Credentials`.
+
+   .. image:: media/google-auth-credentials-button.png
+      :align: center
+      :alt: Credentials button menu
+
+#. Click on *Create Credentials* and select **OAuth client ID**.
+
+   .. image:: media/google-oauth-client-id.png
+      :align: center
+      :alt: Oauth client id selection
+
+#. Select **Web Application** as the Application type. Now configure the allowed pages on which you 
+   will be redirected. 
+   
+   In order to achieve this, in the **Authorized redirect URIs** field, enter your database's domain 
+   immediately followed by ``/auth_oauth/signin``. For example: 
+   ``https://mydomain.odoo.com/auth_oauth/signin``, then click on *Create*.
+
+   .. image:: media/google-create-oauth-client-id.png
+      :align: center
+      :alt: Creating oauth client id
+
+.. _google/auth-odoo:
+
+Google Authentication on Odoo
+-----------------------------
+
+.. _google/client-id:
+
+Retrieve the Client ID
+~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have done the previous steps, two infos are received: Client ID and Client Secret. 
+
+.. image:: media/google-auth-secret-ids.png
+   :align: center
+   :alt: Secret ids display
+
+.. _google/odoo-activation:
+
+Odoo activation
+~~~~~~~~~~~~~~~
+
+*Fill out* your Client ID into the **General Settings** on Odoo (under **Integrations**).
+	
+.. image:: media/odoo-settings-client-id.png
+   :align: center
+   :alt: Filling out the client id in odoo settings
+
```

Comment:
and then, what about the settings for each user?
![image](https://user-images.githubusercontent.com/46810843/116225313-fa645100-a751-11eb-8dbe-ee0a4c020ac2.png)